lib/orogen/gen/templates.rb

```diff
@@ -9,10 +9,22 @@ class << self
 	    attr_reader :templates
 	end
 
-        # Returns the directory where Orogen's lib part sits (i.e. where
-        # autobuild.rb and autobuild/ are)
+        # Returns the directory where oroGen's lib part sits, i.e.
+        # base_dir is defined by the directory where orogen.rb resides
         def self.base_dir
```

Comment:
not missing anything but my confused state of mind ;), so sticking with moving the templates (and leaving out the change of base_dir, except for the documentation change)